pages/common/tldr.md

```diff
@@ -1,20 +1,20 @@
 # tldr
 
-> Displays simple help pages for command-line tools, from the tldr-pages project.
+> Display simple help pages for command-line tools from the tldr-pages project.
 > More information: <https://tldr.sh>.
 
-- Show the tldr page for a command (hint: this is how you got here!):
+- Print the tldr page for a specific command (hint: this is how you got here!):
 
 `tldr {{command}}`
 
-- Show the tldr page for `cd`, overriding the default platform:
+- Print the tldr page for a specific subcommand:
 
-`tldr -p {{android|linux|osx|sunos|windows}} {{cd}}`
+`tldr {{command}}-{{subcommand}}`
 
-- Show the tldr page for a subcommand:
+- Print the tldr page for a command for a specific [p]latform:
 
-`tldr {{git-checkout}}`
+`tldr {{-p}} {{android|linux|osx|sunos|windows}} {{command}}`
```

Comment:
Since when do we put flags in placeholder syntax?
```suggestion
`tldr -p {{android|linux|osx|sunos|windows}} {{command}}`
```

Reply:
It's my mistake here. Thanks! :)